cypress/e2e/create_tx.cy.js

```diff
@@ -103,10 +144,11 @@ describe('Queue a transaction on 1/N', () => {
       cy.get('input[type="checkbox"]').should('not.exist')
     })
 
-    cy.contains('Submit').click()
+    // We don't want to keep queing tx for now
+    //cy.contains('Submit').click()
   })
 
-  it('should click the notification and see the transaction queued', () => {
+  it.skip('should click the notification and see the transaction queued', () => {
```

Comment:
We should undo this change if are queueing the created transaction as this test validates the notification.

Reply:
Since I disable the tx proposal I skipped the notification test as well. If we all agree to have tx keep being queued I will remove the skip